src/Reflection/Adapter/ReflectionUnionType.php

```diff
@@ -5,26 +5,27 @@
 namespace Roave\BetterReflection\Reflection\Adapter;
 
 use ReflectionUnionType as CoreReflectionUnionType;
-use Roave\BetterReflection\Reflection\ReflectionIntersectionType as BetterReflectionIntersectionType;
-use Roave\BetterReflection\Reflection\ReflectionNamedType as BetterReflectionNamedType;
+use Roave\BetterReflection\Reflection\ReflectionType as BetterReflectionType;
 use Roave\BetterReflection\Reflection\ReflectionUnionType as BetterReflectionUnionType;
 
-use function array_filter;
 use function array_map;
+use function assert;
 
 final class ReflectionUnionType extends CoreReflectionUnionType
 {
     public function __construct(private BetterReflectionUnionType $betterReflectionType)
     {
     }
 
-    /** @return array<ReflectionNamedType> */
+    /** @return list<ReflectionNamedType|ReflectionIntersectionType> */
```

Comment:
Can we make this a `non-empty-list`?

Reply:
This was marked as resolved, I think? Not feasible?

Reply:
I was just blind. Fixed.